src/build-scripts/gh-installdeps.bash

```diff
@@ -66,9 +66,10 @@ if [[ "$ASWF_ORG" != ""  ]] ; then
         icpx --version
     fi
 
-else
+elif [[ "$USE_CONAN" != "1" ]] ; then
     # Using native Ubuntu runner
 
+    sudo add-apt-repository ppa:ubuntu-toolchain-r/test
```

Comment:
I don't remember why I did these changes. We could probably remove them I guess...

Reply:
Gone